packages/engine/tests/units/state/empty/object/src/behaviors/test.js

```diff
@@ -0,0 +1,6 @@
+/**
+ * Verifies the type of the object
+ */
+const behavior = (state, context) => {
+  state.o1_is_struct = typeof state.o1 === "object";
```

Comment:
I think this test could use some improving, JS sucks and this isn't really checking it's a serialised arrow struct (i.e. a standard JS object)

https://stackoverflow.com/a/8511350/14687716

Reply:
Can you create a follow-up task in the backlog please

Reply:
[Link to Asana task](https://github.com/hashintel/hash/pull/299#discussion_r805808833) (_internal_)